gpkit/nomials/nomial_math.py

```diff
@@ -469,6 +451,25 @@ def __init__(self, left, oper, right):
         self.nomials = [self.left, self.right, self.p_lt, self.m_gt]
         self.nomials.extend(self._unsubbed)
 
+    def simplify_posy_ineq(self, exps, cs):
+        "Simplify a posy <= 1 by moving constants to the right side."
+        coeff = 1.0
+        exps_ = []
+        nonzero_exp_ixs = []
+        for i, exp in enumerate(exps):
+            if exp:
+                nonzero_exp_ixs.append(i)
+                exps_.append(exp)
+            else:
+                coeff -= mag(cs[i])
+        if len(exps_) < len(exps):
+            if coeff < 0:
+                raise ValueError("infeasible constraint: %s" % self)
+            elif coeff == 0:
+                raise ValueError("tautological constraint: %s" % self)
+            cs = cs[nonzero_exp_ixs]
```

Comment:
this is probably silly, but would it be slightly faster to do a `if coeff > 0` case first, for the standard case, and then move into the cases that raise ValueErrors?